recaplog fails to create daily summaries on RHEL 6

I've noticed that on RHEL 6, recap 2.0.2 fails to create the daily summaries with the following error:

```
2019-04-10 01:00:01+01:00 [INFO] --- Starting recaplog[22870] ---
2019-04-10 01:00:01+01:00 [INFO] recaplog (22870): Created lock file: /var/lock/recaplog.lock
2019-04-10 01:00:01+01:00 [INFO] Compressing old log files
2019-04-10 01:00:01+01:00 [INFO] Finding reports from: 20190409
2019-04-10 01:00:01+01:00 [INFO] Reports found: [  ]
2019-04-10 01:00:01+01:00 [ERROR] Unable to archive unexisting reports.
2019-04-10 01:00:01+01:00 [INFO] Deleting log files older than 15 days...
2019-04-10 01:00:01+01:00 [INFO] Deleting: 432 log files.
2019-04-10 01:00:01+01:00 [INFO] Deleting: 0 empty directories.
2019-04-10 01:00:01+01:00 [INFO] recaplog (22870): Caught signal - deleting /var/lock/recaplog.lock
2019-04-10 01:00:01+01:00 [INFO] --- Ending recaplog[22870] ---
```

The issue seems to be that the awk regex to retrieve the report names it's not interpreted correctly. I've applied the following patch to resolve it but I haven't tested it on any other OS:

```
*** recaplog-old        2019-04-10 09:32:43.027843716 +0100
--- recaplog    2019-04-10 09:33:07.242570939 +0100
***************
*** 133,139 ****
    log INFO "Finding reports from: ${day}"
    reports=(
      $( ls -1 "${BASEDIR}" 2>/dev/null |
!          awk -F. \
           '
             /'${day}'-[0-9]{6}\.log$/ {sub("_'${day}'-[0-9]{6}","",$1); print $1}
           ' |
--- 133,139 ----
    log INFO "Finding reports from: ${day}"
    reports=(
      $( ls -1 "${BASEDIR}" 2>/dev/null |
!          awk --posix -F. \
           '
             /'${day}'-[0-9]{6}\.log$/ {sub("_'${day}'-[0-9]{6}","",$1); print $1}
           ' |
```

--- a/src/recaplog
+++ b/src/recaplog
@@ -133,7 +133,7 @@ get_report_names_of_day() {
   log INFO "Finding reports from: ${day}"
   reports=(
     $( ls -1 "${BASEDIR}" 2>/dev/null |
-         awk -F. \
+         awk --posix -F. \
          '
            /'${day}'-[0-9]{6}\.log$/ {sub("_'${day}'-[0-9]{6}","",$1); print $1}
          ' |

